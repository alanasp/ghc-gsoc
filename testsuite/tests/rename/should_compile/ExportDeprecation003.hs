{-# OPTIONS_GHC -fwarn-warnings-deprecations #-}

-- Test deprecation of variable and data constructor exports

module ShouldCompile where

import ExportDeprecation003_A

foo = bfunc
