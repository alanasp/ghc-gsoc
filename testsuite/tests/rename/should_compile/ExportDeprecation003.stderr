
ExportDeprecation003.hs:9:7: warning: [-Wdeprecations (in -Wdefault)]
    In the use of ‘bfunc’
    (imported from ExportDeprecation003_A, but defined in ExportDeprecation003_B):
    Deprecated: "module ExportDeprecation003_B reexport is deprecated"
